src/olympia/landfill/serializers.py

```diff
@@ -0,0 +1,185 @@
+                total_reviews=777,
+                weekly_downloads=22233879,
+                developer_comments='This is a testing addon.',
+            )
+            Preview.objects.create(addon=addon, position=1)
+            Review.objects.create(addon=addon, rating=5, user=user_factory())
+            Review.objects.create(addon=addon, rating=3, user=user_factory())
+            addon.reload()
+
+            addon.save()
+            print(
+                'Created addon {0} for testing successfully'
+                .format(addon.name))
+
+    def create_addon(self):
+        """Creates a custom addon named 'Ui-Addon'.
+
+        This addon will be a featured addon and will have a featured collecton
+        attatched to it. It will belong to the user uitest.
+
+        It has 1 preview, 5 reviews, and 2 authors. The second author is named
+        'ui-tester2'. It has a version number as well as a beta version.
+
+        """
+        default_icons = [x[0] for x in icons() if x[0].startswith('icon/')]
+        addon = addon_factory(
+            status=STATUS_PUBLIC,
+            type=ADDON_EXTENSION,
+            average_daily_users=10000,
+            users=[UserProfile.objects.get(username='uitest')],
+            average_rating=5,
+            description=u'My Addon description',
+            file_kw={
+                'hash': 'fakehash',
+                'platform': amo.PLATFORM_ALL.id,
+                'size': 42,
+            },
+            guid=generate_addon_guid(),
+            icon_type=random.choice(default_icons),
+            name=u'Ui-Addon-Test',
+            public_stats=True,
+            slug='ui-test-2',
+            summary=u'My Addon summary',
+            tags=['some_tag', 'another_tag', 'ui-testing',
+                  'selenium', 'python'],
+            total_reviews=1000,
+            weekly_downloads=9999999,
+            developer_comments='This is a testing addon.',
+        )
+        Preview.objects.create(addon=addon, position=1)
+        Review.objects.create(addon=addon, rating=5, user=user_factory())
+        Review.objects.create(addon=addon, rating=3, user=user_factory())
+        Review.objects.create(addon=addon, rating=2, user=user_factory())
+        Review.objects.create(addon=addon, rating=1, user=user_factory())
+        addon.reload()
+        AddonUser.objects.create(user=user_factory(username='ui-tester2'),
+                                 addon=addon, listed=True)
+        version_factory(addon=addon, file_kw={'status': amo.STATUS_BETA},
+                        version='1.1beta')
+        addon.save()
+        generate_collection(addon, app=FIREFOX)
+        print(
+            'Created addon {0} for testing successfully'
+            .format(addon.name))
+
+    def create_theme(self):
+        """Creates a custom theme named 'Ui-Test Theme'.
+
+        This theme will be a featured theme and will belong to the uitest user.
+
+        It has one author.
+
```

Comment:
The method should be renamed a bit to reflect it's more full-featured implementation, something like `create_featured_theme_with_data` or so?